lib/helper/Puppeteer.js

```diff
@@ -1069,6 +1071,12 @@ class Puppeteer extends Helper {
       fs.mkdirSync(downloadPath, '0777');
     }
     fsExtra.emptyDirSync(downloadPath);
+    // Backwards compatibility with Puppeteer < 14.4.0
+    if (parseInt(version.split('.')[0], 10) >= 14) {
```

Comment:
can we just check for >= 15 ?

also I propose a better syntax to handle version:
```js
const [majorVersion] = version.split('.');
if (majorVersion >= 15)  ...
```

Reply:
I'm just wanting to do the backwards compatibility, like 14.3 and backwards should be not broken. So do you think we only care the major version, don't you?

Reply:
yes, 15.x is current one, we should not bother of some minor like 14.x